simtools/corsika/corsika_output.py

```diff
@@ -708,17 +787,19 @@ def get_2D_num_photons_distr(self):
         Returns
         -------
         numpy.array
-            Number of photons per event per telescope.
+            Number of photons per event per telescope in self.telescope_indices.
```

Comment:
I removed the parameters description, since they were reminiscent of previous implementations.

The function returns the distribution (histogram) of the number of photons in various events. It is important to give one an idea of how many events are necessary, for a specific CORSIKA configuration, to reach a normal distribution. We will in the future want to compare distributions for different inputs but we want to be sure that the number of events simulated is enough for the comparison.

`get_2D_num_photons_distr` is different than the other histogram functions. While the others will lead to a number of histograms, depending on the flags `individual_telescopes` and `telescope_indices`, `get_2D_num_photons_distr` always returns the same shape. The list added in `visualize` is a trick to allow to use the same for loop for all the distributions. in this sense, all distributions are lists.